typescript-sdk/integrations/agno/examples/agent.py

```diff
@@ -29,4 +30,6 @@
 app = agui_app.get_app()
 
 if __name__ == "__main__":
-  agui_app.serve(app="agent:app", host="0.0.0.0", port=9001, reload=True)
+  port = int(os.getenv("PORT", "9001"))
```

Comment:
Were these docs out of date and you're just updating?

Reply:
The docs _forced_ running on port 9001. Now it looks for a PORT environment variable, and if that's not present, defaults to 9001. Just makes it easier to run agents locally if you can configure their port (so we can avoid collisions)